### PR TITLE
Add guide for `importing-inject-from-ember-service` deprecation

### PR DESCRIPTION
**Pull request by bertdeblock**
https://github.com/emberjs/ember.js/pull/20526

### CODE REVIEW

**Comment by IgnaceMaes** [comment]
Content looks good, but I believe we'll need a new folder for Ember v6 deprecations.

**Comment by bertdeblock** [comment]
Woops, didn't notice the v5 folder when reopening. Made this one a while ago.

File: content/ember/v6/importing-inject-from-ember-service.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+Importing `inject` from `@ember/service` is deprecated. Please import `service` instead.
+
+Example:
+
+```diff
+import { Route } from '@ember/routing/route';
+- import { inject as service } from '@ember/service';
++ import { service } from '@ember/service';
```
Comment (IgnaceMaes) [inline content/ember/v6/importing-inject-from-ember-service.md:15]:
```suggestion
```js {data-filename="ember-cli-build.js" data-diff="-2,+3"}
import { Route } from '@ember/routing/route';
import { inject as service } from '@ember/service';
import { service } from '@ember/service';
```

**Comment by IgnaceMaes** [comment]
Why are the tests failing 🤔

**Comment by mansona** [comment]
The tests are failing because of the node tests (specifically because of the markdown validation): 

```
  1) Json Tests
       ember/v6/importing-inject-from-ember-service.md is ok:
     Error: You don't need to define a displayId if it's the same as the file name
      at Context.<anonymous> (file:///home/runner/work/deprecation-app/deprecation-app/node-tests/index.mjs:23:15)
      at process.processImmediate (node:internal/timers:478:21)
```

File: content/ember/v6/importing-inject-from-ember-service.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Importing `inject` from `@ember/service`
+until: 7.0.0
+since: 6.3.0
+displayId: importing-inject-from-ember-service
+---
```
Comment (mansona) [inline content/ember/v6/importing-inject-from-ember-service.md:6]:
```suggestion
since: 6.3.0
---
```

**Review by IgnaceMaes — APPROVED** [review]

**Comment by IgnaceMaes** [comment]
Aha thanks, I missed the failed test from the logs above. (as the last one did succeed)

**Comment by IgnaceMaes** [comment]
And thanks @bertdeblock for the contribution 😄

**Pull request merged**